src/vs/workbench/services/workingCopy/common/resourceWorkingCopy.ts

```diff
@@ -143,6 +143,15 @@ export abstract class ResourceWorkingCopy extends Disposable implements IResourc
 
 	//#endregion
 
+	//#region Modified Tracking
+
+	// Overridden when the Working Copies need to differentiate between
+	// dirty and modified e.g. scratchpads
+	isModified(): boolean {
+		return this.isDirty();
```

Comment:
Do we still need this when we adopt `isModified()` to return `this.isDirty()` in most places?

Reply:
I think the override is fine but I was wondering about the explicit comment you made there, why do you have it? I think we can remove it, if a subclass wants to change this behaviour, they can.